machine_learning_hep/plot_ml_std_comp.py

```diff
@@ -1,333 +0,0 @@
-#############################################################################
-##  © Copyright CERN 2018. All rights not expressly granted are reserved.  ##
-##                 Author: Gian.Michele.Innocenti@cern.ch                  ##
-## This program is free software: you can redistribute it and/or modify it ##
-##  under the terms of the GNU General Public License as published by the  ##
-## Free Software Foundation, either version 3 of the License, or (at your  ##
-## option) any later version. This program is distributed in the hope that ##
-##  it will be useful, but WITHOUT ANY WARRANTY; without even the implied  ##
-##     warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.    ##
-##           See the GNU General Public License for more details.          ##
-##    You should have received a copy of the GNU General Public License    ##
-##   along with this program. if not, see <https://www.gnu.org/licenses/>. ##
-#############################################################################
-
-"""
-main script for doing final stage analysis
-"""
-import os
-from math import sqrt
-# pylint: disable=import-error, no-name-in-module, unused-import
-import yaml
-from ROOT import TFile, gStyle, gROOT
-from machine_learning_hep.utilities_plot import plot_histograms
-
-FILES_NOT_FOUND = []
-
-# pylint: disable=too-many-branches
-def compare_ml_std(case_ml, ana_type_ml, period_number, filepath_std, scale_std=None,
```

Comment:
Hi @benedikt-voelkel. This function was obsolete? Because you only copied "compare_ml_std_ratio"

Reply:
Exactly, was the same thing. I don't know why we duplicated that. Rush times I guess...